src/config.ts

```diff
@@ -197,11 +197,17 @@ export class KubeConfig {
             scheme = 'http';
         }
 
+        // Wrap raw IPv6 addresses in brackets.
+        let serverHost = host;
+        if (host && host.indexOf(':') != -1) {
+            serverHost = `[${host}]`;
+        }
+
         this.clusters = [
             {
                 name: clusterName,
                 caFile: `${pathPrefix}${Config.SERVICEACCOUNT_CA_PATH}`,
-                server: `${scheme}://${host}:${port}`,
+                server: `${scheme}://${serverHost}:${port}`,
```

Comment:
Maybe we should also be building up the address with Uri helpers? Maybe they can also handle this natively?

Reply:
I couldn't tell from docs if they handled this or not, so I left it as is.